modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/kinky_shocker.dm

```diff
@@ -118,7 +118,7 @@
 		to_chat(user, span_danger("[target] doesn't want you to do that."))
 		return
 	deductcharge(cell_hit_cost)
-	playsound(loc, 'sound/weapons/taserhit.ogg', 70, 1, -1)
+	play_lewd_sound(loc, 'sound/weapons/taserhit.ogg', 70, 1, -1)
```

Comment:
I really wish I knew what all of these `1, -1` stood for.

Edit: I checked, all of the `1` (not `-1`) should actually be `TRUE`, it's a boolean. Not going to force you to do it in this PR though.

Reply:
Yeah, I'll probably want to do that as a future PR